test/test_autograd.py

```diff
@@ -3204,6 +3204,361 @@ def backward(ctx, gO):
                     out.backward()
             self.assertIn('MyFunc.apply', str(w[0].message))
 
+    @unittest.skipIf(not torch.cuda.is_available(), "CUDA unavailable")
+    @skipIfRocm
```

Comment:
Why must this be skipped on ROCm?

Reply:
I guess it doesn't have to be.  But (like i said in https://github.com/pytorch/pytorch/pull/26512#discussion_r327197870) I'm not sure if I should target Rocm with this PR.  AMD gpus don't have tensor cores that internally accumulate in FP32. It's possible their math libraries use pure FP32 math in the actual kernels, but I'm not sure. Whitelist ops that are safe on Volta/Turing may not be safe on rocm, or I may have to reduce the accuracy tolerances of the tests for rocm.

If the situation is that "if i'm targeting cuda, I can't help but target rocm" then I can enable the tests for rocm and see what happens.  I have no way to test rocm locally.

Reply:
Not targeting ROCm for now seems fine.